ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -395,6 +394,50 @@ def test_cumprod(
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.cummin",
+    dtype_x_axis_castable=_get_castable_dtype(),
+    reverse=st.booleans(),
+)
+def test_cummin(
+    *,
+    dtype_x_axis_castable,
+    reverse,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    input_dtype, x, axis, castable_dtype = dtype_x_axis_castable
+    # ToDo: set as_variable_flags as the parameter generated by test_cummin once
+    # this issue is marked as completed https://github.com/pytorch/pytorch/issues/75733
+    if "torch" in backend_fw.__name__:
+        assume(not test_flags.as_variable[0])
+        assume(not test_flags.test_gradients)
+    # gradient tests have been disabled for cummin as the gradients computed by the
+    # backends are inconsistent with tensorflow returning a zero gradient when the
+    # product is zero (discrete optimization), and torch and jax returning a non-zero
+    # gradient based on the value used to compute the product even if it's zero
+    # ToDo: Revisit this later
+    if np.abs(np.min(np.abs(x[0])) - 0) < 1e-4:
+        assume(not test_flags.test_gradients)
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=[input_dtype],
+        test_flags=test_flags,
+        fw=backend_fw,
+        fn_name=fn_name,
+        on_device=on_device,
+        x=x[0],
+        axis=axis,
+        reverse=reverse,
+        dtype=castable_dtype,
+        rtol_=1e-1,
+        atol_=1e-1,
```

Comment:
An absolute and relative tolerances of `1e-1` seems to be a bit high, is there a specific reason behind this?

Reply:
In my opinion, we are not doing any arithmetic operations in the cummin function that could lead to different values in different framework. Therefore a high tolerance seem acceptable. What do u think about it?